### ISSUE TITLE
M> Create Brand Platform Document

### ISSUE THREAD

**Issue by pmoorman** [rchain/bounties#371]
_Pre-P.S. this issue is spun out of the #234 marketing audit_

---

**Problem:**
The seems to be growing unrest within the coop about the branding. Many discussions in #exec-committee and #members-only revolve around branding issues. Some clarity is needed.

Also, more and more people come onboard through the Bounty Program (or otherwise) to help spread the word about RChain. This is great... but they need some guidelines to represent RChain properly, and stay on-brand. 

Finally, @patrick727 asked me — based on the marketing audit I did before — to compile all the thoughts on branding so-far, and make it ready for an external branding agency to expand on. It is important that the document also takes into account the relationships to partners such as Holdings, Ventures & Pyrofex.

**Scope:** 
Compile all the key brand components into a single document, get feedback within the community, and prepare it for an external branding agency to iterate on the visual components.

_NOTE: the scope does not include the visual assets, as they already have their own place in the brand guidelines. From this document, we link to relevant external sources_

**Tasks / Deliverables:** 
- [x] Define branding needs together with @AyAyRon-P 
- [x] Compile all previous branding work done  (taglines, concepts, ELI5, core story, principles, "start with why". etc.)
- [x] Establish brand identity (naming / spelling) for master and sub brands
- [x] Establish taglines for all entities
- [x] Create brand backstory
- [x] Create (draft) mission statement
- [ ] Discuss mission statement & backstory with members
- [x] Establish brand character & tone-of-voice
- [ ] Create compressed brand platform for Rholang
- [ ] Create compressed brand platform for the Cooperative
- [ ] Create appendixes for key partners: Holdings, Ventures, Pyrofex (so we can unify the JV brandings)
--
- [ ] Facilitate members discussion(s)... which will probably be numerous :100: 
- [ ] Finalise, and make presentable for @patrick727 so he can use it well going forward

---

_NOTE: We already started working on this together with @AyAyRon-P. To avoid a flood of opinions in a pre-mature stage, we'll get a draft ready before we push it out to the community._

_We'll wait a little for the Governance Forum, to see what comes out of that, and try integrate that immediately into the document._ 

**Budget: $5000**

**Comment by AyAyRon-P** [comment]
Looking at the doc, I think we can remove the brand guidelines for RhoLang and have an "about" page for it was a short description. The brand guidelines will be in the master style guide which is yet to be produced.

**Comment by drbloom** [comment]
[![Foo](http://www.coop/wp-content/uploads/2017/05/coop_blk-300x126.png)]

.COOP Global Cooperative Marque branding was discussed and **agreed upon** during the Co-op's January 26 staff meeting. Full adoption of the Marque was applied for and granted to the Cooperative to use under these [Terms & Conditions](http://www.domains.coop/support/terms-conditions/).

### coop Marque

[Coop_marque.zip](http://www.coop/wp-content/uploads/2017/08/Coop_marque.zip) (2.31 MB)

### coop Marque with slogan

[English_slogan.zip](http://www.coop/wp-content/uploads/2017/08/English_slogan.zip) (2.66 MB)

### coop Marque with key messages

[English\_key\_messages.zip](http://www.coop/wp-content/uploads/2017/08/English_key_messages.zip) (19.21 MB)

### coop Marque Custom

[English_custom.zip](http://www.coop/wp-content/uploads/2017/08/English_custom.zip) (5.07 MB)

### Signature images – large

[jpg\_hr\_1.zip](http://www.coop/wp-content/uploads/2017/08/jpg_hr_1.zip) (16.05 MB)[jpg\_hr\_2.zip](http://www.coop/wp-content/uploads/2017/08/jpg_hr_2.zip) (22.96 MB)

### Signature images – medium

[jpg_mr.zip](http://www.coop/wp-content/uploads/2017/08/jpg_mr.zip) (29.61 MB)

### Signature images – small

[jpg_lr.zip](http://www.coop/wp-content/uploads/2017/08/jpg_lr.zip) (7.39 MB)

**Comment by AyAyRon-P** [comment]
@drbloom this isn't the place for that information. That would be something to include in the brand style guide.

**Comment by dckc** [comment]
@pmoorman @AyAyRon-P please see feedback from the telegram audience compiled by @Mervyn853  in https://github.com/rchain/bounties/issues/291#issuecomment-369296566 .

Let us know when you've seen it (e.g. with a reaction on this comment).

**Comment by makys** [comment]
@pmoorman this is good start. I made some comments in the document. Having a consistent way of doing things will help unify us and reduce the barrier for entry for new members.

**Comment by AyAyRon-P** [comment]
Taking a look now at Mervyn's comments and will leave feedback there.

**Comment by AyAyRon-P** [comment]
@dckc I commented on issue #291 

👍

**Comment by drbloom** [comment]
Sigh. Jan 26 staff meeting, attended by all Co-op officers had no minutes. @patrick727 was there as well, and can confirm. After the recent logo confusion, I understand your hesitation, so I'll also check in on this with the Exec-Committee.

**Comment by AyAyRon-P** [comment]
This will change the brand platform and brand identity completely if we are to move forward with full adoption of .COOP and not as an alignment tool.

**Comment by dckc** [comment]
Note also:

- [RChain Maketing Plan Draft - 1][1]

and discussion in [#marketing-launch-plan-proposal-feedback][2].

[1]: https://docs.google.com/document/d/1tfoZMB7tedcsqZG6enGsCN4qR6l6AZSCUNrdMUs_0c4/edit
[2]: https://discordapp.com/channels/375365542359465989/418481658497466379?jump=418597911039967233

**Comment by patrick727** [comment]
I'm don't mean to muddy the water, but my take away from that meeting was less absolute consensus on the .coop and more of vague "that seems like a good idea" type of comments. 

I actually don't see if fitting seamlessly into a logo or branding scheme and doesnt seem like every major co-op is doing that. even rei uses there own stylized co-op for there own branding and doesnt adhere to the full adoption requirements. 

[..braces for impact]

**Comment by drbloom** [comment]
@patrick727 - What do you mean by saying that you don't want to "muddy the water"?
If you've changed your mind since our meeting, please just be **honest** and say so.

(The following is transcribed from the Jan 26 staff meeting)

**Ian:**
> With the branding, I wanted to ask you all, if eveyone is on-board with the idea of doing a co-branding with the .COOP marquis. RChain.coop is registered as a coop with CLUSA and the ICA, the International Cooperative Alliance and so that's how we got our .COOP.

> They offer a branding guideline for a pretty "COOP" that helps tell the world our identity as a Co-op, and it is something that if we are doing any future branding or logo design, we want to coordinate so that whatever we do might visually match up and go along nicely with this Co-op branding.
 
> Is that an identity that we want to really pursue, or are we looking to be kind of our our own sort of coop? How closely do we want to align with the greater Co-op movement in the world?

**Greg:**
> **I am totally in favor of this. This is a good idea. We want to align ourselves with Co-ops, so, very, very good.**

**Ian:**
> OK, check.

**Pat:**
> To that point.. Yea I was also wondering about, a ..
**first of all I agree to everything**, and I am
I am finalizing some of my action items to send to them today and one of them is that branding firm thing...
Also do we want to like, since we are all RChain, do we want Holdings and Ventures or Reflective and Pyrofex to be considered in this branding also?

**Greg**
> Why don't we just ask them?

**Pat**
> Alright.
I know from a PR-Story  type of standpoint, all parties are on-board, so I'll follow up with that specific question.

**Greg**
> That would be great.
